src_testbed/testbed.rs

```diff
@@ -1172,9 +1123,10 @@ impl State for Testbed {
                         }
 
                         self.set_world(w.3, w.4, w.5);
-                        self.physics.broad_phase = w.1;
-                        self.physics.narrow_phase = w.2;
-                        self.state.timestep_id = w.0;
+                        self.harness.physics.broad_phase = w.1;
+                        self.harness.physics.narrow_phase = w.2;
+                        //FIXME: not completely sure this is valid
+                        self.harness.state.timestep_id = w.0;
```

Comment:
```suggestion
                        self.harness.state.timestep_id = w.0;
```
This should be valid, yes.